client/web/compose/src/components/PageBlocks/MetricConfigurator/index.vue

```diff
@@ -203,10 +207,14 @@
                 :label="$t('metric.edit.transformFunctionLabel')"
                 label-class="text-primary"
               >
-                <b-form-textarea
+                <c-input-expression
                   v-model="edit.transformFx"
+                  auto-complete
+                  lang="javascript"
                   placeholder="v"
                   class="mb-1"
+                  height="55.16px"
```

Comment:
can the heights be in rem please. We try not to use px since the base size is defined in px and then rem is relative to that.